.github/actions/silences-validate/silences-validate.sh

```diff
@@ -30,6 +30,13 @@ valid_until_date() {
       return 1
     fi
     echo "  [ok] valid until correct $valid_until"
+          if gh pr view "$(git branch --show-current)" --json number >/dev/null 2>&1; then
```

Comment:
Beware of the indentation.

Also, this test deserves some comments:
```suggestion
          # don't check if the date falls on a week-end when no PR is attached
          if gh pr view "$(git branch --show-current)" --json number >/dev/null 2>&1; then
```